source/atomics.tex

```diff
@@ -427,6 +427,7 @@
 
 \rSec1[atomics.types.generic]{Atomic types}
 
+\indexlibrary{\idxcode{atomic<\placeholder{type}>}}%
```

Comment:
Replaced all uses of atomic<type> with just atomic, where appropriate to that type. Added fresh entries for atomic<integral> and atomic<T*>, to better index their larger interfaces. Then removed any remaining references to atomic<type>.